### ISSUE TITLE
Error: SHA256 mismatch while installing with Homebrew

### ISSUE THREAD

**Issue by core45** [caskformula/homebrew-caskformula#63]
I have this error while installing:

brew install caskformula/caskformula/inkscape
==> Installing inkscape from caskformula/caskformula
==> Downloading https://launchpad.net/inkscape/0.92.x/0.92.2/+download/inkscape-0.92.2.tar.bz2
==> Downloading from https://launchpadlibrarian.net/332184472/inkscape-0.92.2.tar.bz2
######################################################################## 100.0%
==> Downloading https://gitlab.com/inkscape/inkscape/commit/93ccf03162cd2e46d962822d5507865f3451168c.diff
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 1f037cc29cee8e0c60ab4753d4151741c8170e4849129bac68fdc60925eb971d
Actual: 7ce4f70f1cb03cd0bab7dc94821a89bf71600bd59c763ae0d78c48f33db3d9ac
Archive: /Users/kornel/Library/Caches/Homebrew/inkscape--patch--1f037cc29cee8e0c60ab4753d4151741c8170e4849129bac68fdc60925eb971d.diff
To retry an incomplete download, remove the file above.

**Comment by campbead** [comment]
I have gotten this identical error.  

I deleted both `inkscape--0.92.2.tar.bz2` and 
`inkscape--patch--1f037cc29cee8e0c60ab4753d4151741c8170e4849129bac68fdc60925eb971d.diff` then retried the install and received the same error.

**Comment by hauntingEcho** [comment]
it looks like this is the same as #39

**Comment by ghost** [comment]
I likewise have the identical error. 
I also deleted both inkscape--0.92.2.tar.bz2 and
inkscape--patch--1f037cc29cee8e0c60ab4753d4151741c8170e4849129bac68fdc60925eb971d.diff then retried the install and received the same error.

**Comment by Impavish** [comment]
I have completed all of the above steps and received the same error.

**Comment by davidhi420** [comment]
Also receiving this error, MacOS 10.11

**Comment by poraschaudhary** [comment]
Same here in MacOS 10.13.6

**Comment by qheolet** [comment]
Meet too... No solution?

**Comment by andresovela** [comment]
Same error here.

**Comment by durch** [comment]
The `sha` changed for some, possibly malicious reason, it is possible to update the sha directly in the formula -> `brew edit caskformula/caskformula/inkscape`, however I'd advise against it until we know why it changed. Probably better to go with `.dmg` -> https://inkscape.org/en/release/0.92.2/mac-os-x/107/dmg/dl/

**Comment by ashegan** [comment]
I have the same error on macosx 10.13.6

**Comment by vagari** [comment]
The workaround, alternative install, mentioned on #39 (and meantioned in the readme), worked for me.

`brew install caskformula/caskformula/inkscape --HEAD --branch-0.92`

**Comment by charlesrocket** [comment]
@vagari not working on mojave

```
      dict.dictAdd(key, std::move(obj));
      ~~~~~^~~~~~~
/usr/local/include/poppler/Object.h:276:8: note: candidate function has been explicitly deleted
  void dictAdd(char *key, Object &&val) = delete;
       ^
/usr/local/include/poppler/Object.h:359:21: note: candidate function
inline void Object::dictAdd(const char *key, Object &&val)
                    ^
1 error generated.
make[2]: *** [src/CMakeFiles/inkscape_base.dir/extension/internal/pdfinput/pdf-parser.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [src/CMakeFiles/inkscape_base.dir/all] Error 2
make: *** [all] Error 2

**Comment by Zverik** [comment]
Same for me. Looks like poppler 0.69 introduced incompatible changes.

**Comment by erdie** [comment]
Same with me, same error on macOS Mojave 10.14

**Comment by ndberg** [comment]
Not working on macOS High Sierra either

**Comment by tsherwen** [comment]
I see the same error on OS Mojave 10.14. Installing manually from the .dmg worked (https://inkscape.org/release/0.92.2/mac-os-x/).

**Comment by udeysri** [comment]
I'm having the same issue

**Comment by midi1007** [comment]
`brew install caskformula/caskformula/inkscape --HEAD --branch-0.92`
worked for me on high sierra :)

**Comment by Zverik** [comment]
Yes, thanks for the reminder. [This commit](https://gitlab.com/inkscape/inkscape/commit/6afa35c3d33c050e8d00bf7608733aad5d726d0b) in 0.92.4 mush have fixed the issue. Everyone else, please try installing again.

From the [changelog](https://inkscape.org/release/inkscape-0.92.4/): "Inkscape can now be built with up-to-date poppler library 0.72.0. This is especially useful for Mac OS users, using Homebrew for building Inkscape."

**Comment by shastrula** [comment]
I did this and then did helm update. It worked fine

brew cleanup --prune=0

**Comment by mymistoe** [comment]
My mac OS version is Big Sur 11.2.1 (20D74).
I have the same error while installing:

> brew install wtf
> ==> Downloading https://downloads.sourceforge.net/project/bsdwtf/wtf-20210212.tar.gz
> Already downloaded: /Users/mistoe/Library/Caches/Homebrew/downloads/124f49fa78a97fd61aa2bd13178560a438e1bf990721fbcecf584094d8e2b0e4--wtf-20210212.tar.gz
> Error: SHA256 mismatch
> Expected: 684bf77efcebf9c707c614abbe2551d1272f8781722cea35ad23056e2e8692ca
>   Actual: 77285a6bf6b12684a8b38ea7d2850e7be9f0c8102101f4eb74b92cc08a5f023d
>     File: /Users/mistoe/Library/Caches/Homebrew/downloads/124f49fa78a97fd61aa2bd13178560a438e1bf990721fbcecf584094d8e2b0e4--wtf-20210212.tar.gz
> To retry an incomplete download, remove the file above.


When I tried` brew edit wtf`, to replace SHA256 with 684bf77efcebf9c707c614abbe2551d1272f8781722cea35ad23056e2e8692ca, it didn't work properly. 

Then I ran `brew cleanup --prune=0`, and everything was fine.

**Comment by StefanSchubert** [comment]
Hi,
the thread is somewhat older, but I encountered the same Problem currently with dart-sass-1.56.1.tar.gz

Already downloaded: /Users/schubert/Library/Caches/Homebrew/downloads/ff09d061ebc95d19ee8c15039ffda36de755442aa8f3b373c3d014e864ee6515--dart-sass-1.56.1.tar.gz
Error: SHA256 mismatch
Expected: db510624782f6ef87e04bbc05666e8d8cf2bccd53bd18f4c954b80cda2cb9e9a
  Actual: 4c6622eb1ac1fb4047b101dc0232df0acb7fd604409a13198f4d9b02b675bde9
    File: /Users/schubert/Library/Caches/Homebrew/downloads/ff09d061ebc95d19ee8c15039ffda36de755442aa8f3b373c3d014e864ee6515--dart-sass-1.56.1.tar.gz
To retry an incomplete download, remove the file above.

Dropping the file, trying differnt Internet accesses to sort out an application firewall problem, but I get the same result each time.
